### DISCUSSION TITLE
`PL` Does not Enable `pylint` Rules

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [astral-sh/ruff#14646] [Q&A]
As a point of clarification, setting `PL` in `tool.ruff.lint.select` does not enable `pylint` settings, but also `ruff` does not say `PL` is an `"unknown rule selector"` if used.

Is this intended behavior? In order to get all `pylint` rule settings, I *actually* need to enter the following in my `pyproject.toml`:

```toml
[tool.ruff.lint]
select = [
    ...
    "PLC",  # pylint
    "PLE",
    "PLR",
    "PLW",
    ...
]
```

**Comment by dhruvmanila** [comment]
> As a point of clarification, setting `PL` in `tool.ruff.lint.select` does not enable `pylint` settings, but also `ruff` does not say `PL` is an `"unknown rule selector"` if used.

Can you clarify how do you know that `PL` is not enabling the Pylint rules? Because, I think it should. You can verify that using `ruff check --show-settings .` as seen below:
```console
$ ruff check --show-settings .
[..]
linter.rules.should_fix = [
        type-name-incorrect-variance (PLC0105),
        type-bivariance (PLC0131),
        type-param-name-mismatch (PLC0132),
        single-string-slots (PLC0205),
        dict-index-missing-items (PLC0206),
        iteration-over-set (PLC0208),
        useless-import-alias (PLC0414),
		[..]
]
[..]
```

**Reply by adam-grant-hendry** [reply]
Huh...I think this was a false alarm. I'm not sure what happened, but it works.

**Comment by adam-grant-hendry** [comment]
Closing as false alarm.
